Firebase/Messaging/CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
-# 2019-07 -- v4.1.1
+# 2019-08-08 -- v4.1.2
+- Fix hang when token is not available before topic subscription and unsubscription. (#3438)
```

Comment:
Please add `[fixed]` in front of the test, like in [this change](https://github.com/firebase/firebase-ios-sdk/pull/3462/files#diff-c9e53a229d5101459a3078d18b68a12f). This will make it easier for us to generate reference docs.

Reply:
Done